plugins/obs-filters/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-project(obs-filters)
+﻿project(obs-filters)
```

Comment:
Looks like this is a bug in VS2017. I'll fix all the files by hand using Notepad++.